### ISSUE TITLE
superuser

### ISSUE THREAD

**Issue by dadinugroho** [mdmsoft/yii2-admin#176]
Thank you for great extension. I delay my upgrade to yii2 because I could not find good extension for rbac like auth in yii 1. 

Since I found great extension, I now start to upgrade to yii2. However, I need one more needed feature, superuser. Is that already implemented? 

Terima kasih

**Comment by mindochin** [comment]
I support the idea. you have to manually write checks for admin

**Comment by mdmunir** [comment]
> I need one more needed feature, superuser. Is that already implemented?

Yes. Create role `super-admin`, assign it with route `*`. Assign user with that role, then you got super user.

**Comment by mindochin** [comment]
> Yes. Create role super-admin, assign it with route *. Assign user with that role, then you got super user

in this case, not working "if(Yii::$app->user->can('index/config'))"

**Comment by mdmunir** [comment]
> in this case, not working "if(Yii::$app->user->can('index/config'))"

You are right. Thats limitation from Yii2. Feel free to submit issue to https://github.com/yiisoft/yii2/issues
